src/components/CartItemsList.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { StyledCartProduct } from "../style";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTimes } from "@fortawesome/free-solid-svg-icons";
+
+const CartItemsList = ({ cartItems, handleRemoveFromCart }) => {
+  return (
+    <StyledCartProduct>
+      {cartItems.length === 0 ? (
```

Comment:
I wouldn't assume that cartItems will always be initialised as an array:
```suggestion
      {cartItems && cartItems.length === 0 ? (
```